docs/guides/validation/validate_data_overview.md

```diff
@@ -9,4 +9,48 @@ import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
 
 <UniversalMap setup='inactive' connect='inactive' create='inactive' validate='active'/>
 
-** This is a placeholder until content is created, so that corresponding links don't cause docusaurus to break **
+When you have completed this step for the first time, you will have created and run a Checkpoint.  This Checkpoint can then be reused to Validate data in the future, and you can also create and configure additional Checkpoints to cover different use cases, should you have them.
```

Comment:
Total personal preference nit - the double present perfect strikes me as a little unusual, though I'm not sure if the below is grammatically correct.

```suggestion
When you complete this step for the first time, you will have created and run a Checkpoint.  This Checkpoint can then be reused to Validate data in the future, and you can also create and configure additional Checkpoints to cover different use cases, should you have them.
```